Ficek_QuantiApp/Ficek_QuantiApp/Features/Sources/RocketErrorView/RocketErrorView.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+import ComposableArchitecture
+import SwiftUI
+import UIToolkit
+
+public struct RocketErrorView: View {
```

Comment:
Mas jeste jeden podobnej target, kterej ma ty extensions pro AlertState. Cili, musis k tomu pristupovat koncepcne. Bud das oboje sem a vytvoris z toho vlastne takovou prepouzitelnou feature, nebo budes mit tuhle cast tady a tamtu cast jinde ale musi to spolu nejak logicky hrat...